pkg/trace/config/peer_tags_test.go

```diff
@@ -43,60 +42,6 @@ func TestPreparePeerTags(t *testing.T) {
 }
 
 func TestDefaultPeerTags(t *testing.T) {
-	expectedListOfPeerTags := []string{
-		"_dd.base_service",
-		"amqp.destination",
-		"amqp.exchange",
-		"amqp.queue",
-		"aws.queue.name",
-		"aws.s3.bucket",
-		"bucketname",
-		"cassandra.keyspace",
-		"db.cassandra.contact.points",
-		"db.couchbase.seed.nodes",
-		"db.hostname",
-		"db.instance",
-		"db.name",
-		"db.namespace",
-		"db.system",
-		"grpc.host",
-		"hostname",
-		"http.host",
-		"http.server_name",
-		"messaging.destination",
-		"messaging.destination.name",
-		"messaging.kafka.bootstrap.servers",
-		"messaging.rabbitmq.exchange",
-		"messaging.system",
-		"mongodb.db",
-		"msmq.queue.path",
-		"net.peer.name",
-		"network.destination.name",
-		"peer.hostname",
-		"peer.service",
-		"queuename",
-		"rpc.service",
-		"rpc.system",
-		"server.address",
-		"streamname",
-		"tablename",
-		"topicname",
-		"dns.hostname",
-		"out.host",
-	}
-	actualListOfPeerTags := basePeerTags
-
-	// Sort both arrays for comparison
-	sort.Strings(actualListOfPeerTags)
-	sort.Strings(expectedListOfPeerTags)
-
-	assert.Empty(t, cmp.Diff(toSet(expectedListOfPeerTags), toSet(actualListOfPeerTags)))
-}
-
-func toSet(list []string) map[string]bool {
-	result := make(map[string]bool)
-	for _, l := range list {
-		result[l] = true
-	}
-	return result
+	// Simple test to ensure peer tags are loaded
+	assert.Contains(t, basePeerTags, "db.name")
```

Comment:
Why do we remove the other checks?

Reply:
https://github.com/DataDog/datadog-agent/pull/28230/files#diff-689864b3834218d5bd46597d505f9bba897c143d718bfe917dac912482b017bf

Reply:
This unit test should not be exhaustive but rather that `basePeerTags` is non-empty and has a valid entry. We then do not have to keep this expected list updated as the ini file is updated.